XenAdmin/Controls/Ballooning/HostMemoryRow.cs

```diff
@@ -47,6 +48,13 @@ public HostMemoryRow(Host host)
             this.host = host;
         }
 
+        public HostMemoryRow(Host host, MetricUpdater metricUpdater)

```

Comment:
The `HostMemoryRow(Host host)` constructor in not used anymore, so it can be removed (as it has been replaced by this one)